storage/interceptors/notifications_interceptor.go

```diff
@@ -74,6 +74,21 @@ func (ni *NotificationsInterceptor) OnTxUpdate(h storage.InterceptUpdateOnTxFunc
 		oldPlatformID := ni.PlatformIdProviderFunc(ctx, oldObject)
 		newPlatformID := ni.PlatformIdProviderFunc(ctx, newObject)
 
+		oldObjectLabels := oldObject.GetLabels()
+		newObjectLabels := newObject.GetLabels()
+		updatedObjectLabels := updatedObject.GetLabels()
+
+		if updatedObject.GetType() == types.VisibilityType {
```

Comment:
I would go for separating them as different interceptors. I thought about brokers and I think we can remove labels at all for now, because we don't have labels that the agents need